assets/src/edit-story/components/checklist/checks/pageBackgroundLowTextContrast/component.js

```diff
@@ -35,37 +46,45 @@ const PageBackgroundTextLowContrast = () => {
     () => filterStoryPages(story, pageBackgroundTextLowContrast),
     [story]
   );
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+
+  const { title, footer } = ACCESSIBILITY_COPY.lowContrast;
+
   return (
     failingPages.length > 0 && (
       <ChecklistCard
-        title={__(
-          'Increase contrast between text and background color',
-          'web-stories'
-        )}
-        /* titleProps={{ onClick: () => { perform highlight here } }} */
-        footer={
-          <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-            {__(
-              'Ensure legibility of text and ease of reading by increasing color contrast',
-              'web-stories'
-            )}
-            {
-              //       <Link
-              //         href={'#' /* figure out what this links to */}
-              //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-              //       >
-              //         {'Learn more'}
-              //       </Link>
-            }
-          </Text>
+        title={title}
+        cardType={
+          failingPages.length > 1
+            ? CARD_TYPE.MULTIPLE_ISSUE
+            : CARD_TYPE.SINGLE_ISSUE
         }
-        /*
-        todo thumbnails for pages
+        footer={<DefaultFooterText>{footer}</DefaultFooterText>}
         thumbnailCount={failingPages.length}
-        thumbnail={<>
-            {failingPages.map(() => <Thumbnail />)}
-          </>}
-      */
+        thumbnail={
+          <>
+            {getVisibleThumbnails(failingPages).map((page) => (
+              <Thumbnail
+                key={page.id}
+                onClick={() => {
+                  setHighlights({
+                    pageId: page.id,
+                  });
+                }}
+                type={THUMBNAIL_TYPES.PAGE}
+                displayBackground={
+                  <PagePreview
+                    page={page}
+                    width={THUMBNAIL_DIMENSIONS.WIDTH}
+                    height={THUMBNAIL_DIMENSIONS.HEIGHT}
+                    as="div"
```

Comment:
hey @samwhale I saw [your comment](https://github.com/google/web-stories-wp/pull/8042#discussion_r658149164) in #8042  (sorry for that poor timing on my renaming) and thought I'd respond here. 
setting this as `div` overrides the `PagePreview` which wants these to be buttons, since I stole it from the carousel where the are indeed buttons. So the thumbnail itself is the button and this is just the "background" since we have icons that show up sometimes on top.

<img width="588" alt="Screen Shot 2021-06-24 at 10 50 58 AM" src="https://user-images.githubusercontent.com/10720454/123309680-15490c80-d4da-11eb-818a-a4cb0814982c.png">